ttnn/cpp/ttnn/operations/conv/conv2d/device/optimized_conv_op_program_factory.cpp

```diff
@@ -211,7 +230,19 @@ operation::ProgramWithCallbacks OptimizedConv::create_program(const std::vector<
     if (input_tensor_a.memory_config().is_sharded()) {
         // If conv_reader_indices is passed in, use v2 where we don't generate indices locally
         if (conv_reader_indices.has_value()) {
-            return multi_core_optimized_conv_sharded_v2_(input_tensor_a, input_tensor_b, this->input_tensor_shape, input_tensor_bias, conv_reader_indices, conv_params, output_channels, untilize_out, has_bias, fuse_relu, parallelization_config, block_config, extra_padding_for_32B_alignment, this->use_shallow_conv_variant, transpose_mcast, output_tensor, this->compute_kernel_config, this->enable_act_double_buffer, this->enable_split_reader, this->enable_subblock_padding);
+            const auto& input_tensor_a_shape = this->input_tensor_shape;
+            auto sliding_window_config = sliding_window::SlidingWindowConfig(
```

Comment:
you can use designated initialiation. In this case you don't need to add comments.